schwifty/iban.py

```diff
@@ -66,6 +66,21 @@ def __init__(self, iban, allow_invalid=False):
     def _calc_checksum_digits(self):
         return "{:02d}".format(98 - (numerify(self.bban + self.country_code) * 100) % 97)
 
+    def _calc_it_checksum_char(country_code, bank_code, account_code):
+        bban_it = [1, 0, 5, 7, 9, 13, 15, 17, 19, 21, 2, 4, 18, 20, 11, 3, 6, 8, 12, 14, 16, 10, 22, 25, 24, 23, 27, 28, 26]
+        if (country_code != 'IT'):
+            return ''
+        tmpSum = 0
+        code = (bank_code + account_code)
+        for idx in range(0, len(code)):
```

Comment:
`range` will by default start at `0`, so this is equivalent to `range(len(code))`